### DISCUSSION TITLE
Repository migration

### DISCUSSION THREAD

**Discussion by pablogm1024** [mps-plato/transit_cam#1] [General]
In order to ensure that the project can be transferred without the limitations of individual accounts, I have set up this organization to own the project and transferred it.

@ilyasku I have invited you to the organization as well. Once you accept the invitation, I will make you administrator both of the organization and the project (I could not do this before, because it was a **personal** project).

We will also have to update the documentation to reflect the current location of the repository.

**Comment by ilyasku** [comment]
Thank you, great idea. I can update the documentation, I already started changing the `star_generator.py` slightly, so that it works with python 3.12.. The other two scripts still work well without any modifications btw.
